HanekeDemo/CollectionViewCell.swift

```diff
@@ -1,39 +0,0 @@
-//
-//  CollectionViewCell.swift
```

Comment:
I'm not suggesting the demo stays at that location. It can be anywhere. Feel free to copy it. I believe, in my opinion, you more likely to get passers by contributing to your code, if your code is less cluttered. Assuming, passers by (developers) glance quickly at code, as I do. The less cluttered the project the better no?

Reply:
In this way updates to Haneke cannot be reflected to the demo. This means that 
1. we should maintain not one but 2 repos.
2. everytime we make changes we should ask you to update
3. users that comes in Haneke and shows interests in it should come to your repo and see how it works.

The demo and the playground has to stay here. Feel free to fork the project and improve it, but we cannot decentralise code that makes sense to keep together.